packages/tooling/src/lib/parameters-to-json-schema.js

```diff
@@ -77,15 +82,10 @@ function getBodyParam(pathOperation, oas) {
             if (!prevProp) {
               // If we have no previous prop, then we're processing a top-level title and description on a requestBody.
               delete obj[prop];
-            } else if (
-              prevProp !== false &&
-              prevProps.includes('schemas') &&
-              'components' in oas &&
-              'schemas' in oas.components &&
-              prevProp in oas.components.schemas
-            ) {
-              // If we have a previous prop, but we're parsing the immediate schemas tree in the components object,
-              // then we're processing a title and description on a component schema.
+            } else if (prevProp !== false && prevProps.length === 1) {
+              // If we have a previous prop, but we're parsing the immediate schemas tree (we know this if prevProps
```

Comment:
this stuff is crazy

Reply:
All this just to remove this kind of stuff because RJSF renders it as an orphan block of text:

![Screen Shot 2020-04-30 at 7 54 54 AM](https://user-images.githubusercontent.com/33762/80725508-07efe800-8ab8-11ea-89a5-785001444006.png)

![Screen Shot 2020-04-30 at 7 56 11 AM](https://user-images.githubusercontent.com/33762/80725591-20f89900-8ab8-11ea-8f36-73e633602b9a.png)